README.md

```diff
@@ -84,44 +84,48 @@ Custom Resources require Kubernetes >= 1.7
 
 ### Configuring the CRD
 
-The CRD can be configured via a [ConfigMap](https://kubernetes.io/docs/api-reference/v1.8/#configmap-v1-core)
-that provides a [ControllerConfig](https://github.com/tensorflow/k8s/blob/master/pkg/spec/controller.go) serialized
-as YAML. The config controls how the CRD manages TensorFlow jobs.
+The CRD must be configured properly to work with your specific Kubernetes cluster.
+Since it will be mounting GPU drivers into your pods, the CRD needs to know where to find them on the Kubernetes agents. It also needs to know which environment variable needs to be injected in the pods.
 
-Currently, the most important use for [ControllerConfig](https://github.com/tensorflow/k8s/blob/master/pkg/spec/controller.go)
-is specifying environment variables and volumes that must be mounted from the
-host into containers to configure GPUS.
+If your Kubernetes cluster is running on GCE (GKE) or Azure (ACS, AKS, acs-engine) simply pass the provider name to the helm install (or in `values.yaml`).
 
-The TfJob controller can be configured with a list of volumes that should be mounted from the host into the container
-to make GPUs work. Here's an example [ControllerConfig](https://github.com/tensorflow/k8s/blob/master/pkg/spec/controller.go):
+For **GCE**
+```
+helm install ${CHART} -n tf-job --wait --replace --set cloud=gce
+```
 
+For **Azure**:
+```
+helm install ${CHART} -n tf-job --wait --replace --set cloud=azure
 ```
+
+If the cluster is not hosted on GCE or Azure, you will need specify a custom configuration.
+To do so edit `${CHART}\custom-config.yaml` with your desired settings.
```

Comment:
I think editing the chart  is undesirable because it requires the user download/unpack the chart.

What if instead of passing in a file to helm, the user has to create the config map manually. e.g.

```
kubectl create configmap tf-job-operator-config --from-file=path/to/bar
helm install ${CHART} -n tf-job --wait --replace --set cloud=none
```
if cloud=none then the helm chart doesn't include the tf-job-operator-config defined in the template.

Reply:
Done.
Slight difference, is that you don't need to specify any value for `cloud`. As long as it it neither `gke` nor `azure` the chart will not create a `ConfigMap`.